test/DynamoCoreTests/Engine/EngineControllerTest.cs

```diff
@@ -82,6 +83,7 @@ public void EngineControllerReconcileTraceDataAndNotify()
             //Act
             var codeBlockNodeOne = CreateCodeBlockNode();
             CurrentDynamoModel.AddNodeToCurrentWorkspace(codeBlockNodeOne, true);
+            codeBlockNodeOne.UpdateValue(new UpdateValueParams("Code", "10;"));
```

Comment:
I guess this was triggering a run before your changes in `WorkspaceModel.cs`?

Reply:
Yes